documentation/adr/0002-use-persistent-term-to-store-schedule-state.md

```diff
@@ -0,0 +1,37 @@
+# 2. Use :persistent_term to store schedule state
+
+Date: 2022-01-26
+
+## Status
+
+Accepted
+
+## Context
+
+Asana: https://app.asana.com/0/116562505129567/1201514090100824
+
+The `Schedule.Fetcher` GenServer makes a call to the `Schedule GenServer` (which a
+lot of Skate calls) when it has new state to provide. Usually this works
+smoothly, but sometimes it causes us to log `GenServer Schedule.Fetcher
+terminating` followed by a longer message `** (stop) exited in:
+GenServer.call(Schedule, [...state])`. This is then followed by some logging on
+the `Schedule` side, with a bunch of warnings like `module=Elixir.Schedule
+function=trip error=timeout` in the couple of minutes after the other error.
+
+It is not purely a deadlock, causes a degradation of performance.
```

Comment:
**issue:** I think this is supposed to read "It is not purely a deadlock, **but it** causes..."